### DISCUSSION TITLE
Função para abrir e ler um arquivo epub

### DISCUSSION THREAD

**Discussion by marciovieira1** [frontendbr/forum#1801] [Perguntas e Respostas]
Olá.
Estou com um problema. Preciso abrir um arquivo com extensão epub e importar os dados para minha aplicação web. Uso Javascript e Node, num banco MySQL.
Conhecem alguma forma de fazer isso ou alguma referência para pesquisa?
Muito obrigado!

**Comment by onlurking** [comment]
Que tipo de dados? Consegue dar um caso de uso mais específico? 

O formato epub é um arquivo ZIP com os capítulos em HTML dentro (sim, um e-book é basicamente um site responsivo).

**Comment by marciovieira1** [comment]
Obrigado pela resposta. 
Basicamente na aplicação web, eu tenho de trazer a programação de um treinamento que vem da matriz por meio de um arquivo epub. 
A aplicação pega essas informações e faz uma designação dos responsáveis por cada parte desse treinamento.
